packages/ui-components/src/components/collapser.ts

```diff
@@ -18,6 +20,17 @@ export class Collapser<T extends Widget = Widget> extends Widget {
     this.addClass('jp-Collapse');
     this._header = new Widget();
     this._header.addClass('jp-Collapse-header');
+    if (collapsed) {
+      this._header.addClass(collapsedClass);
+    }
+    this._header.node.appendChild(caretDownIcon.element({
+      className: 'jp-Collapser-icon'
+    }));
+    const titleSpan = document.createElement('span');
+    titleSpan.classList.add('jp-Collapser-title');
+    titleSpan.textContent = widget.title.label;
+    this._header.node.appendChild(titleSpan);
+
     this._content = new Panel();
     this._content.addClass('jp-Collapse-contents');
```

Comment:
Should we change this to use all caps global const instead? See https://github.com/jupyterlab/jupyterlab/pull/14069/files#r1115181206